packages/aila/src/core/AilaServices.ts

```diff
@@ -23,6 +23,7 @@ export interface AilaLessonService {
   readonly hasSetInitialState: boolean;
   applyPatches(patches: string): void;
   initialise(plan: LooseLessonPlan): void;
+  setUpInitialLessonPlan(messages: Message[]): Promise<void>;
```

Comment:
Moves the initial lesson plan setup to the Aila Lesson which makes more sense - resolves the TODO in the Aila class